python/tvm/tir/__init__.pyi

```diff
@@ -0,0 +1,612 @@
+from typing import (
```

Comment:
Hey just wanted to get a bit of context here. Why this file is under `tir` not `script/tir`?

Reply:
Thanks for the prompt review! There is a discussion in which Wuwei and I decided to put `PrimExpr` and `PrimExprWithOp` (for future use) here in `tir/__init__.pyi` instead of `script/tir/__init__.pyi`. Therefore stubs from `script/tir` will need to import `PrimExpr` and `IterVar` etc from `tir`. I am curious about your take on this.

Reply:
For `tvm.tir` package I think adding inline typing where necessary is sufficient. The main reason that we need a stub file for `tvm.script.tir` is that we would like to expose those keywords that do not have an actual implementation

Reply:
this stub is removed